build.sbt

```diff
@@ -16,6 +16,8 @@ scalaVersion in ThisBuild := "2.11.6"
 
 crossScalaVersions in ThisBuild := Seq("2.10.5", "2.11.6")
 
+parallelExecution in Test := false
```

Comment:
Disabled parallel execution because I encounter some weird exception without this line, e.g.,
```
[info] - checkScalaMethodPresenceVerbose *** FAILED ***
[info]   java.lang.ClassNotFoundException: o .lang.scala.Observ
[info]   at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
[info]   at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[info]   at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[info]   at java.lang.Class.forName0(Native Method)
[info]   at java.lang.Class.forName(Class.java:348)
[info]   at scala.reflect.runtime.JavaMirrors$JavaMirror.javaClass(JavaMirrors.scala:500)
[info]   at scala.reflect.runtime.SymbolLoaders$TopClassCompleter.complete(SymbolLoaders.scala:32)
[info]   at scala.reflect.internal.Symbols$Symbol.info(Symbols.scala:1231)
[info]   at scala.reflect.internal.Types$TypeRef.thisInfo(Types.scala:2407)
[info]   at scala.reflect.internal.Types$TypeRef.baseClasses(Types.scala:2412)
```
I guess some reflection APIs are not thread-safe (or a bug?). So if running multiple `CompletenessKit` at the same time, it will crash.

However, since RxScala's unit tests are pretty fast, disabling it hurts nothing.